docs/app-dev/app-architecture.md

```diff
@@ -5,38 +5,33 @@ order: 3
 # Application Architecture Guide
 
 Here we provide a brief guide on the recommended architecture of a
-Tendermint blockchain application.
-
-The following diagram provides a superb example:
-
-![cosmos-tendermint-stack](../imgs/cosmos-tendermint-stack-4k.jpg)
+CometBFT blockchain application.
 
 We distinguish here between two forms of "application". The first is the
 end-user application, like a desktop-based wallet app that a user downloads,
 which is where the user actually interacts with the system. The other is the
 ABCI application, which is the logic that actually runs on the blockchain.
 Transactions sent by an end-user application are ultimately processed by the ABCI
-application after being committed by the Tendermint consensus.
+application after being committed by CometBFT.
 
-The end-user application in this diagram is the [Lunie](https://lunie.io/) app, located at the bottom
-left. Lunie communicates with a REST API exposed by the application.
-The application with Tendermint nodes and verifies Tendermint light-client proofs
-through the Tendermint Core RPC. The Tendermint Core process communicates with
+The end-user application communicates with a REST API exposed by the application.
+The application runs CometBFT nodes and verifies CometBFT light-client proofs
+through the CometBFT RPC. The CometBFT process communicates with
 a local ABCI application, where the user query or transaction is actually
 processed.
 
-The ABCI application must be a deterministic result of the Tendermint
+The ABCI application must be a deterministic result of the CometBFT
 consensus - any external influence on the application state that didn't
```

Comment:
```suggestion
The ABCI application must be a deterministic result of the consensus
engine of CometBFT - any external influence on the application state that didn't
```
Not sure whether the algorithm should be referred to as CometBFT consensus - it is the Tendermint algorithm,